src/ucp/rma/basic_rma.c

```diff
@@ -26,142 +30,242 @@
         return UCS_ERR_INVALID_PARAM; \
     }
 
-UCS_PROFILE_FUNC(ucs_status_t, ucp_put, (ep, buffer, length, remote_addr, rkey),
-                 ucp_ep_h ep, const void *buffer, size_t length,
-                 uint64_t remote_addr, ucp_rkey_h rkey)
+static inline void ucp_rma_request_wait(ucp_request_t *req)
```

Comment:
could be 'ucp_request_wait_uct_comp' - because need to name functions after what they are doing and not how they are used